lib/features/home/content_panel/contents_bottom_panel_view.dart

```diff
@@ -164,4 +168,37 @@ class _ContentsBottomPanelViewState extends State<ContentsBottomPanelView>
       ),
     );
   }
+
+  LocationView checkLocationView(Placemark? infoLocation, LatLng location) {
+    if (infoLocation == null) {
+      return LocationView(
+        locationIconSvg: Assets.images.idLocationIcon,
+        semanticCoordinate: '${location.latitude}, ${location.longitude}',
+        latitude: location.latitude,
+        longitude: location.longitude,
+        borderRadius: BorderRadius.circular(12),
+      );
+    } else {
+      if (infoLocation.name == '' || infoLocation.name == infoLocation.street) {
+        return LocationView(
+          locationIconSvg: Assets.images.idLocationIcon,
+          address:
+              '${infoLocation.street}, ${infoLocation.subLocality}, ${infoLocation.subAdministrativeArea}, ${infoLocation.administrativeArea}, ${infoLocation.country} ',
+          latitude: widget._location.latitude,
+          longitude: widget._location.longitude,
+          borderRadius: BorderRadius.circular(12),
+        );
+      } else {
+        return LocationView(
+          locationIconSvg: Assets.images.idLocationIcon,
+          businessName: widget._infoLocation?.name,
+          address:
```

Comment:
please compute the `business name` and `location address` in `nartus_geocoder` package, and use that string here.